Command/PopulateCommand.php

```diff
@@ -151,7 +151,7 @@ private function populateIndex(OutputInterface $output, $index, $reset, $options
 
         $this->dispatcher->dispatch(IndexPopulateEvent::POST_INDEX_POPULATE, $event);
```

Comment:
Don't we want to move
```
if ($postPopulate) {
             $this->resetter->postPopulate($index);
         }
```

into listener?

Listener was already created in #744, but than removed in 4c87d24fc12d3e08580bd6aecbfdd3b289e316f2